src/Stratis.Bitcoin.IntegrationTests/BlockStore/RetrieveFromBlockStoreSteps.cs

```diff
@@ -62,6 +62,9 @@ private void a_pow_node_running()
             this.node = this.builder.CreateStratisPowNode(this.network);
             this.node.Start();
             this.node.NotInIBD();
+
+            this.maturity = 1;
```

Comment:
Can we set the maturity in `BeforeTest` or even in `private int maturity = 1;` so that it's more visible, or would that not work?